tests/messageRoute.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('message route', async ({ page }) => {
+  await page.goto('http://localhost:3000/home-page');
+  await page.getByRole('link', { name: 'Image of Red Nike Trainers' }).click();
+  await page.getByRole('button', { name: 'MESSAGE' }).click();
+  await page.getByPlaceholder('Type your message here').click();
+  await page.getByPlaceholder('Type your message here').press('CapsLock');
+  await page.getByPlaceholder('Type your message here').fill('Nice shoes');
+  await page.locator('form').getByRole('button').click();
+  await expect(page.getByText('Nice shoes')).toBeVisible();
+});
```

Comment:
Similarly to the previous PR, we should use a `.describe()` block to identify the test suite and `test()` blocks for each feature we are testing.

In this case we can separate this suite into a few different tests I think:

```js
test.describe('/conversations', () => {
  test('can be reached from /home-page', async ({ page }) => {
    // ...
  });
  
  test('displays latest messages in CurrentConversation', async({page})=>{
    // ...
  })

  test('allows user to delete an existing conversation', async ({page})=>{
    // ...
  })
});
```